libs/rtefsutils/src/RteFsUtils.cpp

```diff
@@ -269,15 +269,17 @@ bool RteFsUtils::SetFileReadOnly(const string& path, bool bReadOnly) {
  */
 
   error_code ec;
-  fs::perms perms;
+  const fs::perms write_mask = fs::perms::owner_write | fs::perms::group_write | fs::perms::others_write;
```

Comment:
Make a real compile-time constant here:
```suggestion
  constexpr auto write_mask = fs::perms::owner_write | fs::perms::group_write | fs::perms::others_write;
```